### DISCUSSION TITLE
Prevent footer to be displayed on top of keyboard

### DISCUSSION THREAD

**Discussion by aefn** [quasarframework/quasar#9131] [General - Components / Directives / etc] [answered]
I don't know that I used a correct category.

Dear All,
I was playing with the Layout builder https://next.quasar.dev/layout-builder.
Let say I want to have a layout that has a footer and that footer should stick at bottom always.
When I open the keyboard, the footer jumps on top of the keyboard.
Probably it can be fixed with ccs styling or some kind of JS event handling,
But maybe I am doing something wrong.
I would like to know is there any predefined functionality in QLayout that I can use to achieve my goal?

Just for clarification : 
![1](https://user-images.githubusercontent.com/28761393/116775901-f85a0500-aa65-11eb-94ee-b86df900a81f.png)
![2](https://user-images.githubusercontent.com/28761393/116775904-fa23c880-aa65-11eb-8130-6e1692ebceff.png)

**Comment by hawkeye64** [comment, marked as answer]
@aefn You said

> Let say I want to have a layout that has a footer and that footer should stick at bottom always.

As far as I can tell, it's doing exactly what you requested. When a mobile keyboard comes up, the web view shrinks in height. The footer is above the keyboard because it's adhering to the rule of stay at the bottom of the view.

**Reply by aefn** [reply]
> @aefn You said
> 
> > Let say I want to have a layout that has a footer and that footer should stick at bottom always.
> 
> As far as I can tell, it's doing exactly what you requested. When a mobile keyboard comes up, the web view shrinks in height. The footer is above the keyboard because it's adhering to the rule of stay at the bottom of the view.

Then probably I expressed my intention wrongly. What can I do, if I want the footer is not displayed when the keyboard is open?

**Reply by metalsadman** [reply]
Bind a v-if on the footer to a computed prop that detects if soft keyboard is showing or not.

**Reply by aefn** [reply]
Thanks, I thought that there is some kind of option in QLayout options. 
I really appreciate your fast response.

**Reply by hawkeye64** [reply]
> Then probably I expressed my intention wrongly. What can I do, if I want the footer is not displayed when the keyboard is open?

Yes, what @metalsadman said. I did misunderstand.
